src/components/custom-sdk/template/HMRC_ODX_GDSCheckAnswersPage/index.tsx

```diff
@@ -100,13 +100,18 @@ export default function HmrcOdxGdsCheckAnswersPage(props: HmrcOdxGdsCheckAnswers
       invalidateCache: true
     };
 
+    const dataPageName = isEduStartJourney()
+      ? 'D_GetCYAStepIDByApplication'
+      : 'D_GetCurrentCYAStepID';
```

Comment:
What would we do for other flows? Would we have a switch statement here?

Reply:
D_GetCYAStepIDByApplication will be common for all other journey except chb

Reply:
As Luke has highlighted it might be an idea to add a switch statement here it will help with the maintainability of the code moving forward and make it easier to add new statements as and when needed ie :
const dataPageName = (() => {
Switch (true) {
case is isEduStartJourney():
  return 'D_GetCYAStepIDByApplication'
  }
  case isCHABJourney():... etc

Reply:
added switch